gme/gme.h

```diff
@@ -298,7 +300,7 @@ BLARGG_EXPORT gme_err_t gme_load_data( Music_Emu*, void const* data, long size )
  * @since 0.6.4
  */
 BLARGG_EXPORT gme_err_t gme_load_tracks( Music_Emu* me,
-                                         void const* data, long* sizes, int count );
+                                         void const* data, size_t* sizes, int count );
```

Comment:
API + ABI change: I don't think this is wanted at all.

Reply:
alright, i'll leave those ones alone